cordapp/src/main/kotlin/net/corda/plugins/Utils.kt

```diff
@@ -14,7 +14,13 @@ import kotlin.math.max
  * accessor to the "ext" extension (See: ExtraPropertiesExtension)
  */
 @Suppress("UNCHECKED_CAST")
-fun <T : Any> Project.ext(name: String): T = (extensions.findByName("ext") as ExtraPropertiesExtension).get(name) as T
+fun <T : Any> Project.ext(name: String): T? {
```

Comment:
_Wibble?!_ The `quasar-utils` plugin reads the `quasar_group`, `quasar_version` etc properties using just
```groovy
def quasarGroup = rootProject.hasProperty('quasar_group') ? rootProject.property('quasar_group') : defaultGroup
```
Do you _need_ this extension function?